File: Version_Python/GeradorMatriz.py
import random

def gerar(dim, tipo):
    # Gera a matriz de acordo com o tipo
    m = [[0]*dim for _ in range(dim)]


    # Preenche a matriz
    for i in range(dim):
        for j in range(dim):
            if tipo == 0:
                m[i][j] = 1
            elif tipo == 1:
                m[i][j] = 100000
            else:
                m[i][j] = random.randint(1, 100000)
    return m

File: Version_JavaScript/GeradorMatriz.js
function gerar(dim, tipo) {
    // Gera a matriz de acordo com o tipo
    const m = Array.from({ length: dim }, () => new Array(dim));

    // Preenche a matriz
    for (let i = 0; i < dim; i++) {
        for (let j = 0; j < dim; j++) {
            if (tipo === 0) m[i][j] = 1;
            else if (tipo === 1) m[i][j] = 100000;
            else m[i][j] = Math.floor(Math.random() * 100000) + 1;
        }
    }
    return m;
}

module.exports = { gerar };